x-pack/plugins/screenshotting/server/browsers/chromium/integration_tests/paths.test.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { loggingSystemMock } from '@kbn/core-logging-server-mocks';
+import assert from 'assert';
+import axios from 'axios';
+import del from 'del';
+import path from 'path';
+import { paths as chromiumArchivePaths } from '../../../utils';
+import { download } from '../../download';
+import { md5 } from '../../download/checksum';
+import { install } from '../../install';
+
+/* eslint-disable no-console */
+
+const mockLogger = loggingSystemMock.create().get();
+mockLogger.warn = jest.fn((message: string | Error) => {
+  console.warn(message);
+});
+mockLogger.debug = jest.fn((message: string | Error) => {
+  console.log(message);
+});
+mockLogger.error = jest.fn((message: string | Error) => {
+  console.error(message);
+});
+
+describe('Chromium Archive Paths', () => {
```

Comment:
This test touches on https://github.com/elastic/kibana/issues/155753, but I think we need to keep that issue open. I still think we need a way to see the version of Chromium from a running instance.